src/radical/pilot/configs/resource_xsede.json

```
@@ -552,6 +552,52 @@
         "python_dist"                 : "default"
     },
 
+    "bridges_mpiexec": {
+        "description"                 : "The XSEDE 'Bridges' cluster at PSC (https://portal.xsede.org/psc-bridges/).",
+        "notes"                       : "Always set the ``project`` attribute in the ComputePilotDescription.",
+        "schemas"                     : ["gsissh", "ssh", "go"],
+      # "mandatory_args"              : [],
+        "gsissh"                      :
+        {
+            "job_manager_endpoint"    : "slurm+gsissh://bridges.psc.xsede.org:2222/",
+            "filesystem_endpoint"     : "gsisftp://bridges.psc.xsede.org:2222/"
+        },
+        "ssh"                         :
+        {
+            "job_manager_endpoint"    : "slurm+ssh://bridges.psc.xsede.org/",
+            "filesystem_endpoint"     : "sftp://bridges.psc.xsede.org/"
+        },
+        "go":
+        {
+            "job_manager_endpoint"    : "slurm+ssh://bridges.psc.xsede.org/",
+            "filesystem_endpoint"     : "go://xsede#bridges/"
+        },
+        "default_queue"               : "RM",
+        "resource_manager"            : "SLURM",
+        "lfs_path_per_node"           : "$LOCAL",
+        "lfs_size_per_node"           : 3713368,
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "agent_launch_method"         : "MPIEXEC",
+        "task_launch_method"          : "MPIEXEC",
+        "mpi_launch_method"           : "MPIEXEC",
```

```suggestion
        "agent_spawner"               : "POPEN",
        "agent_launch_method"         : "SSH",
        "task_launch_method"          : "SSH",
        "mpi_launch_method"           : "MPIEXEC",
```

May I suggest this change then? For some reason the agent did not start with MPIEXEC.

---

I would like to keep `MPIEXEC` for the `task_launcher` though, as it does not suffer from saturation issues like `SSH`.

---

That is pushed now.  Having said that, do you understand why MPIEXEC failed for the agent?  Do you still happen to have the logs around?

---

I did not pay too much attention as the purpose was to get repex running. Unfortunately, I do not have the logs around anymore. I can generate them if you want.

---

Nah, don't worry - we have enough on our table...  Lets handle that when it pops up again...